etc/arm.yaml

```diff
@@ -15,6 +15,7 @@ CONTROLLER:
     filter_size: 30
     max_invalid: 5
     thresholds: collision_thresholds.yaml
+  has_floating_base: true
```

Comment:
I would suggest to rename to `floating_base` (`floating_base: true` reads well).

Reply:
agreed, changed both the yaml configuration variable name and the controller class member